edx_name_affirmation/tests/test_models.py

```diff
@@ -1,15 +1,40 @@
 """
 Tests for Name Affirmation models
 """
+from unittest.mock import patch
 
 from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
 from django.test import TestCase
 
 from edx_name_affirmation.models import VerifiedName
 from edx_name_affirmation.statuses import VerifiedNameStatus
 
 User = get_user_model()
 
+idv_attempt_id = 34455
+idv_attempt_status = 'submitted'
+
+idv_attempt_id_notfound = 404
+idv_attempt_id_notfound_status = None
```

Comment:
Nit. Since these variables are only in use in `test_verification_status`, they'd probably make more sense to define in that test method instead of the top of the file.

Let me know if you disagree!

Reply:
The one variable that's used just in `test_verification_status` is `idv_attempt_id_notfound_status`, since `idv_attempt_id_notfound` and `idv_attempt_status` are used in the helper method _mocked_model_get.

I think I tried to define this helper method within `test_verification_status itself()` but the linter complained about it and suggested me to put it at the top level.

I'll try to write it down in a way that gets scoped inside the test method.

Reply:
I moved these variables and the helper methods to be defined within `VerifiedNameModelTests` class so the scope is more limited. Let me know what you think.